ouroboros-consensus/docs/interface-CHANGELOG.md

```diff
@@ -56,6 +56,15 @@ may appear out of chronological order.
 The internals of each entry are organized similar to
 https://keepachangelog.com/en/1.1.0/, adapted to our plan explained above.
 
+## Circa 2021-11-17
+
+### Changed
+
+- The initialization step of the ChainDB is now running unmasked, so it is
+  possible to stop the node while it is initializing the ChainDB with the usual
+  means (async exceptions). No changes are needed downstream to integrate this
```

Comment:
I suppose it's not harmful, but because "No changes are needed downstream to integrate this change", I don't think we _need_ to update this file.

Reply:
I was exactly going to ask you for your opinion on today’s meeting. I’m okey with not adding it, will remove the commit and then merge